core/domain/stats_services.py

```diff
@@ -47,14 +46,11 @@ def _migrate_to_latest_issue_schema(exp_issue_dict):
         Exception. The issue_schema_version is invalid.
     """
     issue_schema_version = exp_issue_dict['schema_version']
-    if issue_schema_version is None or issue_schema_version < 1:
```

Comment:
Are you sure `issue_schema_version` can never be None?

Reply:
Reverted